webrender_api/src/api.rs

```diff
@@ -171,6 +175,10 @@ impl Transaction {
         self.merge(resources);
     }
 
+    pub fn notify(&mut self, event: ExternalEvent) {
```

Comment:
can we have  some documentation here on what does this mean? When will the notification be fired, and why we need it.